File: DataSummitSolution/DataSummitUI/src/app/efModels/category.ts

export class Category {

	CategoryId: number;
	Name: string;
	Description: string;
	Picture: string;
	CreatedDate?: Date | string;
	UserId?: number;

	constructor(categoryId?: number, name?: string, description?: string,
		picture?: string, createdDate?: Date | string, userId?: number)
	{
		this.CategoryId = categoryId || 0;
		this.Name = name || "";
		this.Description = description || "";
		this.Picture = picture || "";
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.UserId = userId || 0 || null;
	}
}

File: DataSummitSolution/DataSummitModels/DB/Category.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class Category
    {
        public int CategoryId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public byte[] Picture { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }
    }
}
